src/dinit.cc

```diff
@@ -453,6 +471,90 @@ static int process_commandline_arg(char **argv, int argc, int &i, options &opts)
     return 0;
 }
 
+#if SUPPORT_SELINUX
+// Load the system SELinux policy and transition ourselves to it. When successful,
+// this will cause SELinux labels as per the policy to be attached to processes (and
+// file descriptors owned by those processes). The SELinux framework will begin to
+// enforce restrictions on access based on these labels and the loaded policy.
+//
+// We might lose access to any file descriptors we have open when this is called (since
+// they will still be labelled with the kernel context), so it is best done early (i.e.
+// before we start opening file descriptors).
+//
+// Parameters:
+//   exe - the path that we are invoked with (to calculate our new security
+//         context to tranition into.)
```

Comment:
Typo here: tranition -> transition

Reply:
Sorted in https://github.com/davmac314/dinit/pull/400/commits/10c8198efc4477d59ad466d74f91956bc2a2580d